README.md

```diff
@@ -31,6 +31,7 @@ To create a star chart for tonight's sky as seen from [Palomar Mountain](https:/
 ```python
 from datetime import datetime
 from pytz import timezone
+from starplot.map import Projection
```

Comment:
ah good catch!

this isn't well-documented yet, but `Projection` is also part of the base `starplot` package, so instead of importing `Projection` here, could you change line 39 below to `projection=sp.Projection.ZENITH,` ? thanks!

Reply:
ha! that's the way I had it first, then changed it! can do